src/Illuminate/Bus/Dispatcher.php

```diff
@@ -129,19 +129,21 @@ protected function marshal($command, ArrayAccess $source, array $extras = [])
 	 * @return mixed
 	 */
 	protected function getParameterValueForCommand($command, ArrayAccess $source,
-                                                   ReflectionParameter $parameter, array $extras = array())
+                                                   ReflectionParameter $parameter, array $extras = [])
 	{
 		$value = $this->extractValueFromExtras($parameter, $extras);
-		
+
 		if (is_null($value))
 		{
 			$value = $this->extractValueFromSource($source, $parameter);
 		}
-		elseif (is_null($value) && $parameter->isDefaultValueAvailable())
+
+		if (is_null($value) && $parameter->isDefaultValueAvailable())
 		{
 			$value = $parameter->getDefaultValue();
 		}
-		elseif (is_null($value))
+
+		if (is_null($value))
```

Comment:
@darryldecode Feel free to send a pull request you my fork. 

https://github.com/GrahamForks/framework/tree/5.0-bus